WooCommerce/WooCommerceTests/ViewRelated/Blaze/BlazeAddParameterViewModelTests.swift

```diff
@@ -0,0 +1,105 @@
+import XCTest
+@testable import WooCommerce
+
+final class BlazeAddParameterViewModelTests: XCTestCase {
+    func test_when_entering_view_with_no_parameters_then_save_button_disabled() {
+        // Given
+        let sut = BlazeAddParameterViewModel(
+            remainingCharacters: 100,
+            parameter: nil,
+            onCancel: { },
+            onCompletion: { _, _ in }
+        )
+
+        // Then
+        XCTAssertTrue(sut.shouldDisableSaveButton)
+    }
+
+    func test_when_entering_view_with_existing_parameters_then_save_button_enabled() {
```

Comment:
Non-blocking suggestion: should we check for changes to enable the Save button when entering view with existing paramaters?

Reply:
I agree! I'll work on a new PR for this. I played around with the logic needed and found it can be a little complex.